src/styles/partials/components/_summary-list.scss

```diff
@@ -0,0 +1,30 @@
+.dg_summary-list {
+  background-repeat: no-repeat;
```

Comment:
lines 2 and 3 are not impacting this list, because there is no background image. You can set background color should be added to `dg_summary-list` to it looks like `$gray-lightest`

Reply:
done